packages/prover/src/cli/cmds/start/options.ts

```diff
@@ -33,35 +36,42 @@ export const startOptions: CliCommandOptions<StartArgs> = {
     choices: ["rest", "p2p"],
   },
 
-  "beacon-urls": {
+  beaconUrls: {
     description: "The beacon node PRC urls for 'rest' mode.",
     type: "string",
     array: true,
     demandOption: false,
   },
 
-  "beacon-bootnodes": {
+  beaconBootnodes: {
     description: "The beacon node PRC urls for 'p2p' mode.",
     type: "string",
     array: true,
     demandOption: false,
   },
 
-  "ws-checkpoint": {
+  wsCheckpoint: {
     description:
       "The trusted checkpoint root to start the lightclient. If not provided will initialize from the latest finalized slot. It shouldn't be older than weak subjectivity period",
     type: "string",
   },
+
+  unverifiedWhitelist: {
```

Comment:
Just curious, is there a specific reason why this is not handled as an array? 

The pattern I usually see is that if a list of values is allowed we use `array` which gives the option to provide the flag multiple times or alternatively space-separated values.
This would work by default and if you also want to support comma-separated values, could add custom `coerce` function as we do here
https://github.com/ChainSafe/lodestar/blob/7c8e5f8e31e29bdd78e6c66551919819a2dd07d7/packages/cli/src/cmds/validator/options.ts#L280

Reply:
I didn't knew `yargs` allows space separated values as flag options. Normally that's not the case for a Linux CLI interfaces. Any value without a flag is considered as argument not as flag options. 

I will double check and if it is then update the PR.